R/model.R

```diff
@@ -0,0 +1,21 @@
+#' @title Run the simulation
+#' @description
+#' The main entrypoint for the simulation. run_simulation puts together the
+#' model components and runs the malaria simulation. This currently returns a
+#' dataframe with the number of individuals in each state at each timestep
+#'
+#' @param pop population. See \code{\link[squire]{get_population}}
```

Comment:
We've got roxygen+markdown working here so you can do
```
See [squire::get_population]
```

and this will set up links for you (same with other cross-links)